### ISSUE TITLE
[DetailsList]: [Details header Header] Aria attribute "aria-sort" is not invalid for the button elements. 

### ISSUE THREAD

**Issue by sulabh-msft** [OfficeDev/office-ui-fabric-react#1017]
### Bug Report

- __Package version(s)__: 1.9.0
- __Browser and OS versions__: Internet Explorer/Windows 10

#### Describe the issue:
In DetailsList , if details list header is specified 
"aria-sort" should be applied to the element having role="columnHeader". Currently it is applied to the button element for which ari-sort is invalid attribute.

https://dequeuniversity.com/rules/axe/2.1/aria-allowed-attr

```html
<div class="ms-DetailsHeader-cellWrapper" role="columnheader">
    <button disabled="" class="ms-DetailsHeader-cell history-message-header" aria-haspopup="false" aria-label="Message" aria-sort="none" data-automationid="ColumnsHeaderColumn" data-item-key="message" style="width: 699px;">Message</button>
</div>
```

**Comment by dzearing** [comment]
@cschlechty known issue? Does this affect screen readers? If so we should just fix! Seems super easy.

@sulabh-msft If you'd like to submit a small PR for this fix, that would be awesome.

**Comment by sulabh-msft** [comment]
This has been fixed as part of #970 .
We can close this issue as duplicate of #970
